lawnchair/src/app/lawnchair/gestures/handlers/SleepGestureHandler.kt

```diff
@@ -63,12 +63,11 @@ class SleepGestureHandler(context: Context) : GestureHandler(context) {
 }
 
 class SleepMethodRoot(context: Context) : SleepGestureHandler.SleepMethod(context) {
-    private val rootHelperManager = RootHelperManager.INSTANCE.get(context)
 
-    override suspend fun isSupported() = rootHelperManager.isAvailable()
+    override suspend fun isSupported() = Shell.getShell().isRoot
```

Comment:
Why don't use `rootHelperManager.isAvailable()` here?

Reply:
it's not working anymore for some reason, I'm not sure tho what's causing it

Reply:
Also already tried to fix the current dt2s but this what I get, instead of make it sleep it just redirect to lockscreen

**Using old implementation :** 

https://github.com/LawnchairLauncher/lawnchair/assets/36076410/7f16a463-32aa-4bb9-84dd-80bf8f1a6787

Reply:
https://github.com/LawnchairLauncher/lawnchair/blob/7aa0b7172c75000bfd3e113cc009f52dab9f979a/lawnchair/src/app/lawnchair/root/RootHelperManager.kt#L27

Try
```kt
    private val isAvailableDeferred by lazy {
        scope.async(Dispatchers.IO) {
            Shell.getShell().isRoot
        }
    }
```

Reply:
I think it's better to drop it, why access it if we can simplify it like the latest commit one?

Reply:
Not sure why, I guess it might be a blocking job, better to put it on IO?

Reply:
No, it will work by changing IO to Main, but if we gonna use `Shell.su("input keyevent 26").exec()` for DTS2 no need to put in RootHelperManager or you want to keep the current one, you will get output like the screen record I send above